internal/gcsx/temp_file.go

```diff
@@ -332,15 +332,15 @@ const (
 func (tf *tempFile) ensure(limit int64) error {
 	switch tf.state {
 	case fileIncomplete:
-		size, err := tf.f.Seek(0, 2)
+		size, _ := tf.f.Seek(0, 2)
```

Comment:
Should we return err instead of ignoring?

Reply:
In certain io methods, an EOF is returned as error. So, an error might not always imply a problem. Therefore I'm a bit scared changing this logic as it might lead to behavior changes. WDYT?